### PR TITLE
Plugin registry housekeeping

### PR DESCRIPTION
**Pull request by ericwill**
Signed-off-by: Eric Williams <ericwill@redhat.com>

### What does this PR do?
* Remove old didact extension
* Delete redhat/php plugin as we have the latest one under the publisher name
* Move redhat/php-debug to the proper publisher/plugin-name folder

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
General maintenance, no issue attached.

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by benoitf** [comment]
should we have aliases for the old names ?
like http redirect ?
also need to be sync with devfile registry ( if still using old artifacts)

**Comment by ericwill** [comment]
> should we have aliases for the old names ?
> like http redirect ?
> also need to be sync with devfile registry ( if still using old artifacts)

Sure, it's a task for Monday :smile:

**Comment by ericwill** [comment]
Aliased the redhat/php-debugger plugin and also made a corresponding devfile PR here: https://github.com/eclipse/che-devfile-registry/pull/301

File: v3/plugins/redhat/php-debugger/latest.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 1.13.0
+
```
Comment (JPinkney) [inline v3/plugins/redhat/php-debugger/latest.txt:2]: Should there be a line here?

File: v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+type: VS Code extension
+description: This VS Code extension provides support for PHP debugging, based on the PHP Debug extension from Felix Backer.
+icon: https://raw.githubusercontent.com/felixfbecker/vscode-php-debug/v1.13.0/images/logo.png
+repository: https://github.com/felixfbecker/vscode-php-debug
+category: Language
+firstPublicationDate: "2019-04-16"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-php:7-b95f23f"
+      name: php-debugger
```
Comment (JPinkney) [inline v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml:16]: I wonder if this needs a memory limit

File: v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+publisher: felixfbecker
+name: vscode-php-debug
+version: 1.13.0
+displayName: PHP Debugger
+title: PHP Debugger
+type: VS Code extension
+description: This VS Code extension provides support for PHP debugging, based on the PHP Debug extension from Felix Backer.
```
Comment (JPinkney) [inline v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml:8]:
```suggestion
description: This VS Code extension provides support for PHP debugging, based on the PHP Debug extension from Felix Becker.
```

File: v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml (hunk 2 above)
Comment (ericwill) [reply v3/plugins/felixfbecker/vscode-php-debug/1.13.0/meta.yaml:16]: What's the policy re: memory limits? I see the devfiles that use this plugin have one set.

**Review by benoitf — APPROVED** [review]

**Pull request merged**